lib/i18n/tasks/interpolations.rb

```diff
@@ -32,10 +32,27 @@ def inconsistent_interpolations(locales: nil, base_locale: nil) # rubocop:disabl
         end
       end
 
+      result.merge!(unrestored_interpolations(locales: locales))
       result.each { |root| root.data[:type] = :inconsistent_interpolations }
       result
     end
 
+    def unrestored_interpolations(locales: nil)
+      locales ||= self.locales
+      result = empty_forest
+
+      locales.each do |locale|
+        data[locale].key_values.each do |key, value|
+          next unless value.is_a?(String)
+          next unless value.index('!!!!!')
```

Comment:
Any reason not to use `#include?` ? It indicates the intent better I think

Reply:
@barak-neeman Can you add this change, before I do my next review?

Reply:
> @barak-neeman Can you add this change, before I do my next review?

@tom-brouwer-bex my bad, I amended the commit without any changes... It's done now.